packages/flutter/test_fixes/material/theme_data.dart

```diff
@@ -237,4 +237,11 @@ void main() {
 
   // Changes made in https://github.com/flutter/flutter/pull/131455
   ThemeData themeData = ThemeData.copyWith(useMaterial3: false);
+
+  // Changes made in https://github.com/flutter/flutter/pull/155072
+  ThemeData themeData = ThemeData();
+  themeData = ThemeData(dialogBackgroundColor: Colors.orange);
```

Comment:
I see the dart fix also provide a case when both dialogBackgroundColor and dialogTheme are not null. Maybe we can also provide a test case here:)

Reply:
Great catch!